pkg/gauges/contracts/LiquidityGaugeFactory.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "@balancer-labs/v2-solidity-utils/contracts/helpers/Authentication.sol";
+import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/Clones.sol";
+import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/Ownable.sol";
+import "@balancer-labs/v2-vault/contracts/interfaces/IVault.sol";
+
+import "./interfaces/ILiquidityGauge.sol";
+
+contract LiquidityGaugeFactory is Authentication, Ownable {
```

Comment:
Do we actually need `Ownable`? We're not using `onlyOwner` anywhere.

Reply:
This is to avoid essentially copying all of `Ownable` into this contract in order to track the AuthorizerAdaptor (we can just make it the owner of this contract.)

If it's causing confusion however we can just copy code across.

Reply:
Ahh I understand now - Ownable is essentially storage for an address plus events.

I think I'd still do it manually instead of using Ownable, as Ownable also carries the ownership semantics, and implies the owner can do things other accounts cannot, which is not the case here (at least not as directly as `onlyOwner`).